e2e/cypress/integration/channel_settings/channel_header_spec.js

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {getRandomId} from '../../utils';
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Channel Settings', () => {
+    let testTeam;
+    let user1;
+    let admin;
+
+    before(() => {
+        cy.apiGetMe().then(({user: adminUser}) => {
+            admin = adminUser;
+
+            cy.apiInitSetup({loginAfter: false}).then(({team, user}) => {
```

Comment:
nit: Not logging in after is the default so I suggest to not pass such value. We'd like to be explicit only when we wanted to login the newly created user on init.
```suggestion
            cy.apiInitSetup().then(({team, user}) => {
```